### DISCUSSION TITLE
Unable to search for code within file in my organization

### DISCUSSION THREAD

**Discussion by danish-kamal-sophos** [community/community#129061] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I have created a classic PAT and performed org SSO flow on it. I am trying to do a code search with the following curl, but I am unable to get any results:

```
curl --location 'https://api.github.com/search/code?q=apistore%20org%3Asophos-internal%20path%3A*.properties' \
--header 'Accept: application/vnd.github+json' \
--header 'X-GitHub-Api-Version: 2022-11-28' \
--header 'Authorization: Bearer ********'
```

I do get the results when I remove the `path` filter. Please help

**Comment by danish-kamal-sophos** [comment, marked as answer]
Fixed using below query:

`apistore in:file org:sophos-internal extension:properties`

**Reply by nrrm1** [reply]
Goman
